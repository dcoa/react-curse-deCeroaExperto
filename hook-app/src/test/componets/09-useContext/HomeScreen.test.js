const { mount } = require("enzyme")
const { HomeScreen } = require("../../../components/09-useContext/HomeScreen");
const { UserContext } = require("../../../components/09-useContext/UserContext");

describe('Test on HomeScreen componet', () => {
    // Para ver homescreen
    const wrapper = mount(
        <UserContext.Provider
            value={{
                user: {
                    name: 'Jhon Doe',
                    email: 'example@fff.com'
                }
            }}>
            <HomeScreen />
        </UserContext.Provider>
    );

    test('should show correctry', () => {
        expect(wrapper).toMatchSnapshot();
    });

})
