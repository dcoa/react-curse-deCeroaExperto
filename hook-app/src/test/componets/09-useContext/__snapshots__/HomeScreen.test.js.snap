// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test on HomeScreen componet should show correctry 1`] = `
<div>
  <h1>
    Log in
  </h1>
  <hr />
  <pre>
    {
   "name": "Jhon Doe",
   "email": "example@fff.com"
}
  </pre>
</div>
`;
